frontend/explorer/public/js/main.js

```diff
@@ -254,8 +348,28 @@ function appendRawBlock(element, explorerBlock) {
 }
 
 function appendExplorerBlock(element, explorerBlock) {
+	appendNavigationButtons(element, explorerBlock);
+	appendBlockSearchField(element, explorerBlock);
 	appendBlockStatistics(element, explorerBlock);
 	appendBlockMinerPayouts(element, explorerBlock);
 	appendBlockTransactions(element, explorerBlock);
 	appendRawBlock(element, explorerBlock);
 }
+
+//Changes the document title according to the network the page is running on
+window.onload = function() {
+	var networkName = getBlockchainConstants().chaininfo.NetworkName;
+
+	switch(networkName) {
+		case 'testnet':
+			document.title = 'Threefold Chain TESTNET Explorer';
```

Comment:
Please do not use capitals. Also your title is not always correct. You cannot just overwrite the title. You have to replace the `Explorer` part (sub string). Otherwise the first part of the title will become incorrect on some pages.